src/content/en/2019/third-parties.md

```diff
@@ -138,25 +143,25 @@ The remainder of third-party providers also tend to fall into this usage pattern
+Ads, social, hosting, and certain analytics providers represent the largest negative impact on web performance. While hosting providers deliver a majority of a site’s content and will understandably have a larger performance impact than other third-party categories, they also serve almost entirely static sites that demand very little JavaScript in most cases that should not justify the volume of script execution time. The other categories hurting performance though have even less of an excuse. They fill very narrow roles on each page they appear on and yet quickly take over a majority of resources. For example, the Facebook "Like" button and associated social widgets take up extraordinarily little screen real estate and are a fraction of most web experiences, and yet the median impact on pages with social third parties is nearly 20% of their total JavaScript execution time. It is similar for analytics - these libraries do not directly contribute to the perceived user experience, and yet the 90th percentile impact on pages with analytics third parties is 44% of their total JavaScript execution time.
 
-The silver lining of such a small number of entities enjoying such large market share is that a very limited and concentrated effort can have an enormous impact on the web as a whole. Performance improvements at just the top few hosting providers improve 2-3% of _all_ web requests.
+The silver lining of such a small number of entities enjoying such large market share is that a very limited and concentrated effort can have an enormous impact on the web as a whole. Performance improvements at just the top few hosting providers can improve 2-3% of _all_ web requests.
 
 ### Privacy
 
-The abundance of analytics providers and top heavy concentration of script execution raises two primary privacy concerns for site visitors: site owners are more interested in tracking their users than any other third-party use case and a handful of companies receive information on a large swath of web traffic.
+The abundance of analytics providers and top-heavy concentration of script execution raises two primary privacy concerns for site visitors: the main use case of third-parties is for site owners to track their users and a handful of companies receive information on a large swath of web traffic.
```

Comment:
I'm not 100% positive these are interchangeable statements. When I hear "the main use case" I'm thinking it's the *overwhelmingly* most popular reason to use a third party when what I'm trying to say is it's just the most popular and it's a little sad that it is regardless of the margin of victory. If the previous wording didn't get that across I'm definitely open to new versions.

With that extra context, does that change things? If I'm alone in how I feel about the strength of that phrasing, then this works for me! :)

Reply:
I found the previous wording a little confusing and had to read it several times to get what I thought what you were say, hence why I tried to change it to make it clearer. Open to other suggestions here?

Reply:
What about "the largest use case" instead of the "the main use case"?

Reply:
"largest use case" 👍 perfect!